nyc_taxis/operations/default.json

```diff
@@ -961,7 +961,246 @@
       "operation-type": "esql",
       "query" : "FROM nyc_taxis | stats avg(passenger_count)"
     },
+    {
+      "name": "inlinestats_avg_esql_segment_partitioning",
+      "operation-type": "esql",
+      "query" : "FROM nyc_taxis | inlinestats avg(passenger_count)"
```

Comment:
It's giving me OCD to have some keywords uppercase and some lowercase but this is how the existing stats queries were written 😂